run_dir/design/ont_flowcell.html

```diff
@@ -0,0 +1,153 @@
+{% extends 'base.html' %}
+{% block stuff %}
+{% import re %}
+<div id="page_content">
+    <h1>ONT run <span id="page_title">{{ flowcell.get('name') }}</span></h1>
+
+    <div class="row">
+
+      <div class="col-md-4 borderRight" style="width: 500px; margin-right: 120px">
+        <h4 style="margin-top: 20px; margin-bottom: 10px;">Project</h4>
+        <dl class="dl-horizontal">
+          <dt style="text-align: left;">Project ID</dt>				<dd><a href="/project/{{ flowcell.get('project') }}">{{ flowcell.get("project") }}</a></dd>
+          <dt style="text-align: left;">Project name</dt>	    <dd> {{ flowcell.get("project_name") }} </dd>
+        </dl>
+        <h4 style="margin-top: 20px; margin-bottom: 10px;">Time and status</h4>
+        <dl class="dl-horizontal">
+          <dt style="text-align: left;">Run status</dt>				<dd> {{ flowcell.get("TACA_run_status") }} </dd>
+          <dt style="text-align: left;">Start date</dt>				<dd> {{ flowcell.get("start_date") }} </dd>
+          <dt style="text-align: left;">End date</dt>					<dd> {{ flowcell.get("end_date") }} </dd>
+        </dl>
+        <h4 style="margin-top: 20px; margin-bottom: 10px;">MinKNOW identifiers</h4>
+        <dl class="dl-horizontal">
+          <dt style="text-align: left;">Experiment name</dt>  <dd> {{ flowcell.get("experiment_name") }} </dd>
+          <dt style="text-align: left;">Sample name</dt>		  <dd> {{ flowcell.get("sample_name") }} </dd>
```

Comment:
Convert to a css rule since it's used repeatedly.

Reply:
d6eafddef38dd61c0d94629344c647767212d3ad